suites.go

```diff
@@ -48,12 +49,14 @@ func (id SuiteID) Get(dst []byte) (HashToPoint, error) {
 		}
 		e := &encoding{
 			E: E,
-			FieldEncoding: &fieldEncoding{
+			Field: &fieldEncoding{
 				F:   E.Field(),
 				Exp: exp,
 				L:   s.L,
 			},
-			Mapping: m,
+			// TATIANA: what should name be here?
+			ScalarField: field.NewFp("", E.Field().P()),
```

Comment:
This is not the modulus of the scalar field: `E.Field().P()`, it should be the order of the curve.